tools/lib/find_add_class.py

```diff
@@ -93,6 +93,8 @@ def find(fns):
                         html_classes = ['error', 'success']
                     elif fn == 'ui_report.js' and 'status_classes' in line:
                         html_classes = ['alert']
+                    elif fn == 'modal.js':
+                        continue
```

Comment:
why is this needed?  We probably want to have an explicit exclude list like we do with the other JS files.